minecraft_servers/zonecraft/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "server_name": "zonecraft",
+  "nice_name": "ZoneCraft",
+  "direct_ip": "play.zonecraft.es",
+  "server_wildcards": [
+    "%.zonecraft.es",
+	"%.hypixel.es",
+	"%.amamarla.com",
+	"%.banananetwork.es"
+  ],
```

Comment:
Please fix the formatting here (Remove one space from line 6-9, as in our [example manifest file](https://github.com/LabyMod/server-media#example-manifestjson)

Reply:
Fixed!